dist/index.html

```diff
@@ -0,0 +1,239 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Sass Starter</title>
+  <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,700" rel="stylesheet">
+  <!--build:css css/style.css-->
+  <link rel="stylesheet" href="css/style.css">
+  <!--endbuild-->
+</head>
+
+<body>
+  <div class="header">
+    <div class="header__nav">
+      <div class="container">
+        <div class="header__logo f-left">
+          Creative Tim
+        </div>
+        <div class="header__menu header__menu--hamburger f-right">
+          <img src="../images/MENU.png" alt="Menu">
+        </div>
+        <div class="header__menu header__menu--list f-right">
+          <nav>
+            <a href="" class="header__menuItem">COMPONENTS</a>
+            <a href="" class="header__menuItem"><i class="fas fa-cloud-download-alt"></i>DOWNLOAD</a>
```

Comment:
These 2 represents a separate group, it is clear from the distance between these and social media icons in the mock up. So wrap these in another nav or `<ul>`.